src/beamlime/kafka/consumer.py

```diff
@@ -75,9 +75,9 @@ def make_control_consumer(*, instrument: str) -> Generator[kafka.Consumer, None,
     control_config = load_config(namespace='control_consumer', env='')
     kafka_downstream_config = load_config(namespace='kafka_downstream')
     with make_consumer_from_config(
-        topics=['beamlime_control'],
+        topics=['beamlime_commands'],
```

Comment:
Need to update in topic script as well? also, `beamlime_control` appears in `dashboard.py`